catalog/config.js.tmpl

```diff
@@ -5,5 +5,6 @@ window.__CONFIG = {
   signOutUrl: '$SIGN_OUT_URL',
   team: {
     name: '$TEAM_NAME'
-  }
+  },
+  alwaysRequiresAuth: $ALWAYS_REQUIRES_AUTH
```

Comment:
minor: it's about time to alphabetize the JSON keys for `config`

Reply:
But it's JavaScript. What if the code depends on the key order?

J/k, sure